protos/sources/data-node/api/v2/trading_data.proto

```diff
@@ -1260,7 +1250,14 @@ message AggregatedBalanceConnection {
   PageInfo page_info = 2;
 }
 
-// Filter that is used to filter account data by asset ID, party ID, market ID and account type
+// AccountFilter is used to filter account data.
```

Comment:
```suggestion
// Account filter is used to filter account data.
```

Reply:
This has been corrected as well as some other few places, thanks!